board.js

```diff
@@ -1,6 +1,8 @@
 import canvas from "canvas";
 const { Image } = canvas;
 
+const gutterWidth = 15.0; /* Width of the gutter surrounding the map */
```

Comment:
This is the same as the PADDING const being used elsewhere, we should just pass it through

Reply:
Good point, will address